json-guides/retry-timeout.json

```diff
@@ -91,6 +91,64 @@
                 }
             ]
         },
+        {
+            "name": "TimeoutAnnotation",
+            "title": "Adding the @Timeout annotation",
+            "description": [
+                "Now that you've seen the application easily fail when it loads for an indefinite amount of time, let's apply a Timeout policy to to limit the time the request is made to a transaction history and to prevent the page from loading forever.",
```

Comment:
Rather than failing, the page just loads indefinitely when there is some problem on the server end.
> Now that you've seen the application loads indefinitely when... , let's apply...

Reply:
to to -> to